domains/client/domain-executor/src/system_bundle_processor.rs

```diff
@@ -106,12 +106,18 @@ where
             .client
             .header(BlockId::Hash(parent_hash))?
             .map(|header| {
-                let item = AsPredigest::system_domain_state_root_update(StateRootUpdate {
-                    number: parent_number,
-                    state_root: *header.state_root(),
-                });
-
-                Digest { logs: vec![item] }
+                let system_domain_state_root =
+                    AsPredigest::system_domain_state_root_update(StateRootUpdate {
+                        number: parent_number,
+                        state_root: *header.state_root(),
+                    });
+
+                let primary_block_info: sp_runtime::DigestItem =
+                    AsPredigest::primary_block_info((primary_number, primary_hash));
```

Comment:
Should be possible to just use `DigestItem::primary_block_info()` if trait is in scope, might be more readable.

Reply:
the manual type `DigestItem` was added when doing some debugging locally and forgot to remove, but I also using `DigestItem` directly is more straightforward, updated.